src/app/containers/Brand/index.jsx

```diff
@@ -29,6 +66,8 @@ const BrandContainer = props => {
       maxWidth={maxWidth}
       svg={brandSVG}
       url={`/${service}`}
+      scriptLink={scriptLink && scriptLink}
```

Comment:
What is the reason for having two `scriptLink` here?

Reply:
I had previously added it as a check, so that `scriptLink` is never empty. But this is not required.